### PR TITLE
Benchmark

### PR DESCRIPTION
**Pull request by send-on**
Please review index.js file. 

Changes:
- Added http socket max 50
- Added gzip compression
- Changed setex time.

### CODE REVIEW

File: .benchmark/hello.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ config:
   target: 'http://localhost:3004'
   phases:
     - duration: 60
-      arrivalRate: 300
+      arrivalRate: 200
```
Comment (jesrah) [inline .benchmark/hello.yml]: What's the reason for change of rate? Does this relate to the client.setex line in server/index.js?

File: server/index.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
-const newrelic = require('newrelic');
```
Comment (jesrah) [inline server/index.js:1]: Why did you take out this line and just have "require('newrelic')? Was this an error?

File: server/index.js [hunk 3]
```diff
@@ -65,7 +72,7 @@ const getRestaurant  = (req, res) => {
     .then((nearbyData) => {
       results.nearby = nearbyData
       // client.setex(placeId, 60, JSON.stringify(results));
-      client.setex(placeId, 300, JSON.stringify(results));
+      client.setex(placeId, 60*60, JSON.stringify(results));
```
Comment (jesrah) [inline server/index.js:75]: Any tests you can add for this part?

**Review by jesrah — CHANGES_REQUESTED** [review]

File: server/index.js (hunk 2 above)
Comment (send-on) [reply server/index.js:1]: They both work but newrelic does not need to be declared as a variable.

File: server/index.js (hunk 3 above)
Comment (send-on) [reply server/index.js:75]: No tests yet.  TBD

**Pull request merged**